electron/app/locales/en/webui.json

```diff
@@ -644,6 +644,8 @@
   "domain-design-username-header": "Username",
   "domain-design-password-header": "Password",
   "domain-design-secrets-cell-field-name": "Secrets table {{name}} field at row {{position}} with UID {{uid}}",
+  "domain-design-node-selector-label": "Kubernetes node selector",
```

Comment:
Adjacent `*-label` entries are capitalized, should this be "Kubernetes Node Selector" ?